tests/static-volume_test.go

```diff
@@ -107,7 +111,11 @@ var _ = Describe("checkStaticVolume tests", func() {
 				pv.Spec.ClaimRef.ResourceVersion = ""
 				pv.Spec.ClaimRef.UID = ""
 				_, err = f.K8sClient.CoreV1().PersistentVolumes().Update(context.TODO(), pv, metav1.UpdateOptions{})
-				Expect(err).ToNot(HaveOccurred())
+				// We shouldn't make the test fail if there's a conflict with the update request.
+				// These errors are usually transient and should be fixed in subsequent retries.
+				if !errors.IsConflict(err) {
+					Expect(err).ToNot(HaveOccurred())
+				}
```

Comment:
You could do something like this which will just retry transparently https://github.com/kubevirt/containerized-data-importer/blob/35f9f60ac65309c546a217f5bf9c0992ce1100d8/tests/dataimportcron_test.go#L240-L242 https://onsi.github.io/gomega/#category-3-making-assertions-eminem-the-function-passed-into-codeeventuallycode